src/dspace/_rest.py

```diff
@@ -243,6 +243,25 @@ def put_bitstreamregistry(self, data: dict):
 
     # =======
 
+    def fetch_licenses(self):
+        url ='core/clarinlicenses'
```

Comment:
constant? Or it won't be consistent then?

Reply:
I did it as jm before and he didn't use constant